youtube_dl/YoutubeDL.py

```diff
@@ -1073,7 +1073,7 @@ def _build_format_filter(self, filter_spec):
                 '*=': lambda attr, value: value in attr,
             }
             str_operator_rex = re.compile(r'''(?x)
-                \s*(?P<key>ext|acodec|vcodec|container|protocol|format_id)
+                \s*(?P<key>[a-zA-Z0-9._-]+)
```

Comment:
The set of format string meta fields is strictly defined. This must not match arbitrary field.